docs/articles/documentation/recipes/best-practices/speed-up-test-execution.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+title: Create Faster Tests
+permalink: /documentation/recipes/best-practices/speed-up-test-execution.html
+redirect_from:
+  - /documentation/recipes/speed-up-test-execution.html
+---
+
+# Speed Up Test Execution
+
+This section describes ways to decrease the execution time of TestCafe tests. It includes the following sections:
+
+* [Run Tests Concurrently](#run-tests-concurrently)
+* [Run Tests in Headless Browsers](#run-tests-in-headless-browsers)
+* [Use Roles for Login](#use-roles-for-login)
+* [Set Test Speed](#set-test-speed)
+* [Run Tests in Local Browsers](#run-tests-in-local-browsers)
+* [Mock Requests](#mock-requests)
+* [Optimize Your Page Model](#optimize-your-page-model)
+
+## Run Tests Concurrently
+
+Enable concurrent mode to run multiple browser instances simultaneously. This speeds up the testing process at the expense of higher resource consumption.
+
+You can enable concurrency with the [concurrency](../../reference/configuration-file.md#concurrency) configuration file property:
+
+```json
+{
+  "concurrency": 3
+}
+```
+
+In CLI, use the [--concurency](../../reference/command-line-interface.md#-c-n---concurrency-n) option:
+
+```sh
+testcafe --concurency 3 chrome tests/
+```
+
+For more info on concurrency, read [Run Tests Concurrently](../../guides/basic-guides/run-tests.md#run-tests-concurrently).
+
+## Run Tests in Headless Browsers
+
+Headless browsers take less time to initialize because they don't need to render the application. They also enable you to run tests in environments without graphical capabilities, such as CI containers.
```

Comment:
They *can't* render the application for two reasons:
a) The graphics card does the rendering
b) The application itself can not be rendered, because it consists of more than its interface.

CI containers is a nonsensical term. Containers themselves have nothing to do with CI.
```suggestion
Headless browsers take less time to initialize because they lack a GUI. They also enable you to run tests in containers and other environments without graphical capabilities.
```